### PR TITLE
Glob except

### PR DESCRIPTION
**Pull request by AGuliaiev**

### CODE REVIEW

File: src/main/java/org/example/springintro/dto/CreateBookRequestDto.java [hunk 1]
```diff
@@ -9,7 +11,10 @@ public class CreateBookRequestDto {
     private String title;
     private String author;
     private String isbn;
+    @Min(0)
```
Comment (Elena-Bruyako) [inline src/main/java/org/example/springintro/dto/CreateBookRequestDto.java]: Add validation for all required fields
check task description

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
You have the same mistakes as in other PR

File: src/main/java/org/example/springintro/controller/BookController.java [hunk 2]
```diff
@@ -36,7 +38,7 @@ public BookDto getBookById(@PathVariable Long id) {
     }
 
     @PostMapping
-    public BookDto createBook(@RequestBody CreateBookRequestDto requestDto) {
+    public BookDto createBook(@RequestBody @Valid CreateBookRequestDto requestDto) {
```
Comment (Elena-Bruyako) [inline src/main/java/org/example/springintro/controller/BookController.java:41]: add validation to `public BookDto updateBook`

File: src/main/java/org/example/springintro/dto/CreateBookRequestDto.java [hunk 3]
```diff
@@ -1,15 +1,25 @@
...
 public class CreateBookRequestDto {
+    @NotBlank
     private String title;
+    @NotBlank
     private String author;
+    @NotBlank
     private String isbn;
+    @NotNull
+    @Min(0)
     private String price;
```
Comment (Elena-Bruyako) [inline src/main/java/org/example/springintro/dto/CreateBookRequestDto.java]: why price is String?

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
see pr

File: src/main/java/org/example/springintro/dto/CreateBookRequestDto.java [hunk 4]
```diff
@@ -1,15 +1,26 @@
...
 public class CreateBookRequestDto {
+    @NotBlank
     private String title;
+    @NotBlank
     private String author;
+    @NotBlank
     private String isbn;
-    private String price;
+    @NotNull
+    @Min(0)
```
Comment (okuzan) [inline src/main/java/org/example/springintro/dto/CreateBookRequestDto.java:20]: could be just Positive

**Review by okuzan — APPROVED** [review]

**Pull request merged**